client/components/controllers/PreGameMessenger.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+const PreGameMessenger = props => {
+  return (
+    <div className="container">
```

Comment:
We probably should be using the 'container' class at a higher level.  
Nested containers are bad (They will be ~70-100% of the width of a container that itself is 70-100% width)...it quickly reduces rendering space to nothing.